examples/lessons/1/index.js

```diff
@@ -39,17 +37,18 @@ window.webGLStart = function() {
 
       gl.clear(gl.COLOR_BUFFER_BIT | gl.DEPTH_BUFFER_BIT);
       camera.view.id();
+
       //Draw Triangle
       camera.view.$translate(-1.5, 0, -7);
-      program.setUniform('uMVMatrix', camera.view);
-      program.setUniform('uPMatrix', camera.projection);
+      program.uniforms.uMVMatrix(camera.view);
```

Comment:
- Is this really a good choice of API for replacing the setUniform call?
- Shouldn't we use setters and assignment rather than just function call syntax to update a value?
- Was there an issue with keeping the setUniform method?